bpf/accesslog/l24/read_l2.c

```diff
@@ -17,14 +17,18 @@
 
 #include "l24.h"
 #include "../common/data_args.h"
+#include "api.h"
 
-struct netif_receive_skb {
-	unsigned long long pad;
-	void * skbaddr;
-};
+struct trace_event_raw_net_dev_template {
+    struct trace_entry ent;
+    void *skbaddr;
+    unsigned int len;
```

Comment:
I see you have copied all fields in some tracepoint. Is it necessary? If not, I think we can only declare what we need.

Reply:
OK